src/material_data/MaterialProperties.toml

```diff
@@ -70,6 +69,73 @@
     shear_strength = 283e6
     thermal_conductivity = 121 #W/(m K)
 
+[Al-2524]
+    description = """Advanced Fuselage Skin Alloy. 15–20% improvement in fracture toughness and twice fatigue crack growth resistance of 2024-T3."""
+    source = "10.31399/asm.ad.al0389"
+    density = 2780.0 #kg/m3
+    youngs_modulus = 71e9 #Pa
+    shear_modulus = 28e9 #Pa  # Using Al-2024 since not specified
+    poissons_ratio = 0.35
+    YTS = 310e6 #Yield tensile strength
+    UTS = 421e6  #Ultimate tensile strength
+    shear_strength = 283e6
+    thermal_conductivity = 121 #W/(m K) # Using Al-2024 since not specified
+
+[Al-2324-T39]
+    description = """Alloy 2324-T39 is a higher strength version of alloy 2024-T351 and is a purity controlled composition variant of alloy 2024. 
+    It was developed for tension-dominated, fatigue and fracture critical plate applications. The T39 temper, developed through special fabricating 
+    practices, improves both strength and fracture toughness properties over alloy 2024 plate. Alloy 2324-T39 plate is being used successfully on 
+    lower wing skin and center wing box components of commercial aircraft """
+    source = "https://www.lookpolymers.com/polymer_Alcoa-2324-T39-Aluminum-Plate.php"
+    density = 2770.0 #kg/m3
+    youngs_modulus = 72.4 e9#Pa
```

Comment:
How is this space in `72.4 e9` parsed? There's another one below in the definition of Al-7050-T7 that probably also be removed.

Reply:
Fixed that in [this](https://github.com/MIT-LAE/TASOPT.jl/pull/75/commits/d96421c2082fa2293d8735b145c01c46702a33b9) commit

Reply:
Ah, good. I don't know why GitHub specifically wanted to show me the wrong version of that file.